api/src/main/kotlin/io/spine/protodata/render/SourceFile.kt

```diff
@@ -24,16 +24,16 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.protodata.renderer
+package io.spine.protodata.render
 
 import com.google.common.annotations.VisibleForTesting
 import com.intellij.openapi.fileTypes.FileType
 import com.intellij.openapi.fileTypes.FileTypeRegistry
 import com.intellij.psi.PsiFile
 import com.intellij.psi.PsiFileFactory
 import com.sksamuel.aedile.core.cacheBuilder
-import io.spine.protodata.InsertedPoints
-import io.spine.protodata.file
+import io.spine.protodata.ast.file
+import io.spine.protodata.render.SourceFile.Companion.fromCode
```

Comment:
This is suspicious, since it's a new import from this very file, and nothing else is changed.

I have deleted this import in my IDEA, and nothing bad happened. I think, it's just a consequence of excessive "help", provided by IDE. And it is here, because it is used in the documentation.

Please review.